src/Sylius/Bundle/AdminBundle/Resources/config/routing/security.yml

```diff
@@ -30,3 +30,15 @@ sylius_admin_request_password_reset:
     defaults:
         _sylius:
             redirect: sylius_admin_login
+
+sylius_admin_render_password_reset:
```

Comment:
Shouldn't we generate this URL in the reset password email? [Here](https://github.com/Sylius/Sylius/blob/1.12/src/Sylius/Bundle/CoreBundle/Resources/views/Email/adminPasswordReset.html.twig#L41)?

Reply:
Yeah just realized. 😆